islandora_embed.module

```diff
@@ -21,20 +21,20 @@ function islandora_embed_menu() {
     'type' => MENU_CALLBACK,
     'access callback' => TRUE,
   );
-  $items['islandora/embed'] = array(
-    'title' => 'Islandora Embed',
-    'description' => "Interpret the embedding request and send back JSON objects",
-    'page callback' => 'islandora_embed',
-    'type' => MENU_CALLBACK,
-    'access callback' => TRUE,
-  );
-  $items['islandora/embed/maxwidth/%/maxheight/%/url/%'] = array(
-    'title' => 'Islandora Embed',
-    'description' => "Interpret the embedding request and send back JSON objects",
-    'page callback' => 'islandora_embed',
-    'type' => MENU_CALLBACK,
-    'access callback' => TRUE,
-  );
+//  $items['islandora/embed'] = array(
```

Comment:
We probably don't want to leave commented-out sections in. We can always get them back by reverting the change.